examples/manipulation-demo.launch.py

```diff
@@ -32,6 +38,47 @@ def generate_launch_description():
         description="Path to the game launcher executable",
     )
 
+    launch_game_launcher = ExecuteProcess(
+        cmd=[
+            LaunchConfiguration("game_launcher"),
+            "-bg_ConnectToAssetProcessor=0",
+        ],
+        output="screen",
+    )
+
+    def wait_for_clock_message(context: LaunchContext, *args, **kwargs):
+        rclpy.init()
+        node = rclpy.create_node("wait_for_game_launcher")
+        node.create_subscription(
+            Clock,
+            "/clock",
+            lambda msg: rclpy.shutdown(),
+            QoSProfile(depth=1, reliability=ReliabilityPolicy.BEST_EFFORT),
+        )
+        rclpy.spin(node)
+        return None
+
+    # Game launcher will start publishing the clock message after loading the simulation
+    wait_for_game_launcher = OpaqueFunction(function=wait_for_clock_message)
```

Comment:
Nice! We should integrate it with other demos (especially this [launch](https://github.com/RobotecAI/rai/blob/development/src/rai_bringup/launch/sim_whoami_demo.launch.py))! But I think in other PR.

Reply:
Sure, I can try to add it to other demos as well.